leaderelection/leader_election.go

```diff
@@ -43,13 +43,46 @@ const (
 
 	DefaultHealthCheckTimeout = 20 * time.Second
 
+	nameLeaseDuration = "LEASEDURATION"
+	nameRenewDeadline = "RENEWDEADLINE"
+	nameRetryPeriod   = "RETRYPERIOD"
+	nameHealthCheckTimeout = "HEALTHCHECKTIMEOUT"
```

Comment:
Where is this being used?

Reply:
they are used in getTimer(...), except nameHealthCheckTimeout, which is for future and keep the coding style same in this file

Reply:
Sorry what do you mean for future? When will we use it? Can you share some more details?

Reply:
HealthCheckTimeout(DefaultHealthCheckTimeout = 20 * time.Second) is hard coded right now,
if some body think that also need to be configurable,  just change the hard coded value to function call getTimer(nameHealthCheckTimeout), but this is not the issue I met so did not make that change in this package.